test/specs/sObjectsDefinitions.e2e.ts

```diff
@@ -23,6 +23,9 @@ describe('SObjects Definitions', async () => {
   });
 
   step('Check Custom Objects Customer and Product are within objects folder', async () => {
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - Check Custom Objects Customer and Product are within objects folder`
```

Comment:
nit: can we change this to "Check Custom Objects 'Customer__c' and 'Product__c' are within objects folder"

Reply:
nm, looks like this is an existing step